docs/configuration/index.md

```diff
@@ -746,10 +746,10 @@ These Coordinator static configurations can be defined in the `coordinator/runti
 
 |Property|Description|Required?|Default|
 |--------|-----------|---------|-------|
-|`druid.coordinator.period.metadataStoreManagementPeriod`|How often to run metadata management tasks. |No |PT3600S (1 hour)|
+|`druid.coordinator.period.metadataStoreManagementPeriod`|How often to run metadata management tasks in [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) duration format. |No | PT1H|
```

Comment:
The doc CI fails because of `PT1H`. One way to avoid the spell check for such variables is wrapping them with backticks (`` ` ``).

Reply:
Done